VaticanBAV/borget/BAVborget24.xml

```diff
@@ -0,0 +1,540 @@
+                        <title type="incomplete" ref="LIT1873Mangad" />
+                        <explicit xml:lang="gez">ወለመንበረ፡ ስብሐቲሁ፡ ለእግዚአብሔር፡ ዘሐነጸ፡ ጽርሖ፡ ወአልቦ፡ <choice><sic>ዘየአምሮ፡</sic></choice> ዘእንበለ፡ አብ፡ ባሕቲቱ፡ ወዘእንበለ፡ ክርስቶስ፡ ወልዱ፡ ወመንገሱ፡ ቅዱስ<supplied reason="omitted">፡</supplied>ኦመሓሪ፡ መሐረንኢ፡ ወስረይ፡ ኃጢትየ፡ ሊተ፡ ለገብርከ፡ <add place="above">ኃይለ፡ ጊዮርጊስ፡</add> <add place="overstrike">ወልደ፡ ሩፋኤ<supplied reason="omitted">ል፡</supplied></add></explicit>
+
+                        <msItem xml:id="ms_i1.3.1">
+                          <title type="incomplete" ref="LIT1873Mangad#Second" />
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.3.2">
+                          <title type="incomplete" ref="LIT1873Mangad#Third" />
+                        </msItem>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.4">
+                        <incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> በብርሃን፡ ስሙ፡ ለእግዚአብሔር፡ ኢይትፈለጥ፡ ሰማይ፡ ወምድር፡ ፯መላእክት፡ ብርሃናውያን፡ እለ፡ ይነብሩ፡ ታሕተ፡ መንጦላዕቱ፡ ቅዱስ፡</incipit>
+
+                        <msItem xml:id="ms_i1.4.1">
+                          <title type="incomplete" ref="LIT1758Lefafa#FourthChapter"/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.4.2">
+                          <title type="complete" ref="LIT1758Lefafa#FifthChapter"/>
+                        </msItem>
+
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.5">
+                        <title type="complete" ref="LIT1758Lefafa#SixthChapter"/>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.6">
+                        <title type="complete" ref="LIT1758Lefafa#ThirdChapter"/>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.7">
+                        <title type="incomplete" ref="LIT1873Mangad#Sixth"/>
+                      </msItem>
+
+                    </msItem>
+
+                  </msContents>
+
+                  <physDesc>
+                    <objectDesc form="Scroll">
+                      <supportDesc>
+                        <support>
+                          <material key="parchment"/>
+                        </support>
+                        <extent>
+                          <note>Consists of <measure unit="strips">3</measure> strips sewn together.</note>
+                          <dimensions type="outer" unit="mm">
+                            <width>72</width>
+                            <height>1700</height>
+                          </dimensions>
+                        </extent>
+                      </supportDesc>
+                      <layoutDesc>
+                        <layout columns="1" writtenLines="306"/>
+                      </layoutDesc>
+                    </objectDesc>
+                    <decoDesc>
+                      <decoNote xml:id="d1" type="miniature">
```

Comment:
does each miniature have three crosses or these are three decoNotes with one cross each (? )

i know it is a scroll (hard stuff) but maybe some location (or relation to the text break) should be introduced for each decoNote to keep them apart, otherwise it will be printed as

```
miniature:
Three black and red cross with interlaced border above and below.
miniature:
Three black and red cross with decorative border above and below.
miniature:
Three black and red cross with interlaced border with eyes above and below.
```